pkg/scaffold/manager/cmd.go

```diff
@@ -43,44 +43,56 @@ var cmdTemplate = `{{ .Boilerplate }}
 package main
 
 import (
-	"log"
+	"github.com/onsi/ginkgo"
 
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"{{ .Repo }}/pkg/apis"
 	"{{ .Repo }}/pkg/controller"
 )
 
 func main() {
+	// Set logger
+	logf.SetLogger(logf.ZapLoggerTo(ginkgo.GinkgoWriter, true))
```

Comment:
for reviewer:
Should use `GinkgoWriter` here? Or other loggers?

Reply:
this isn't a test.  we shouldn't be using ginkgo.  Just use `ZapLogger`.